src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/offender/HighLight.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.personrecord.client.model.offender
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class HighLight(
+  @JsonProperty("surname")
+  val preferredName: List<String>? = emptyList(),
+) {
+  companion object
+  fun getPreferredNameString(): String? {
```

Comment:
I don't really like the `String` on the end of this method name

Reply:
Will check if Kotlin allows to override the getter.